packages/v-protocol/src/serializer.ts

```diff
@@ -47,8 +47,7 @@ const startup = (opts: Record<string, string>): Buffer => {
     }
     writer.addCString(key).addCString(opts[key])
   }
-  writer.addCString('protocol_version').addInt32(parseInt(opts['protocol_version']))
-  writer.addCString('client_encoding').addCString('UTF8')
+  writer.addCString('protocol_version').addInt32(parseInt(opts['protocol_version'])).addCString('')
```

Comment:
client_encoding is not part of our protocol, leftover from postgres. 

null terminator from the client encoding was allowing the startup request packet to be accepted by 12.0 server

All tests pass locally now against 10.0 server as well, previously gave error for invalid startup packet.

Reply:
probably add a comment for this null terminator hack?